app/client/src/components/pages/Community/components/tabs/EatingFish.js

```diff
@@ -0,0 +1,143 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+// components
+import TabErrorBoundary from 'components/shared/ErrorBoundary/TabErrorBoundary';
+import AssessmentSummary from 'components/shared/AssessmentSummary';
+import WaterbodyList from 'components/shared/WaterbodyList';
+import DisclaimerModal from 'components/shared/DisclaimerModal';
+import ShowLessMore from 'components/shared/ShowLessMore';
+// contexts
+import { LocationSearchContext } from 'contexts/locationSearch';
+// utilities
+import { useWaterbodyFeatures, useWaterbodyOnMap } from 'utils/hooks';
+
+// given a state code like AL and an array of state objects from attains states service,
+// returns the full name of the state
+function convertStateCode(stateCode: string, stateData: Array<Object>) {
+  if (stateData.length === 0) return stateCode;
+  const matchingState = stateData.filter((s) => s.code === stateCode)[0];
+  return matchingState ? matchingState.name : stateCode;
+}
+
+// grammatically correct way of separating list items
+// used to prepend commas/and before a list item
+function addSerialComma(index: number, arrayLength: number) {
+  // first item in list
+  if (index === 0) return '';
+  // not the first or last item in list
+  if (index !== 0 && index !== arrayLength - 1) return ', ';
+  // last item in list
+  if (index === arrayLength - 1) return ' and ';
+}
+
+// --- styled components ---
+const Container = styled.div`
+  padding: 0 1em 1em 1em;
```

Comment:
Since left/right padding is the same, you could use the shorthand notation here:

```suggestion
  padding: 0 1em 1em;
```